keep/api/routes/topology.py

```
@@ -280,3 +290,237 @@ def pull_topology_data(
+    except Exception as e:
+        raise HTTPException(
+            status_code=500, detail=f"Failed to create service: {str(e)}"
+        )
+
+
+@router.put("/service", description="Updating a service manually")
+def update_service(
+    service: TopologyServiceUpdateRequestDTO,
+    authenticated_entity: AuthenticatedEntity = Depends(
+        IdentityManagerFactory.get_auth_verifier(["write:topology"])
+    ),
```

Ruff (0.8.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)